packages/frontend/src/components/create-campaign/preview/index.tsx

```diff
@@ -350,8 +350,15 @@ export function CampaignPreview({
             </Typography>
             <Typography variant="xl2" weight="medium">
                 {safeContext ? t("launched.safe.1") : t("launched.standard")}
-                {safeContext && t("launched.safe.2")}
             </Typography>
+            {safeContext && (
+                <>
+                    <br />
```

Comment:
I think we can avoid the `br` tag here, having a separate `Typography` is enough.